### ISSUE TITLE
Conflicting dependencies

### ISSUE THREAD

**Issue by franva** [espdev/scikit-mpe#9]
The scikit-fmm just fixed a bug in it's latest version 2021.2.2
If you could like to see what's the bug there, here is the link:
https://github.com/scikit-fmm/scikit-fmm/issues/49

So I updated my project requirements.txt to use its latest version, then conflict happened.
Here is the error message:

```
Deploying...
Successfully deployed changes
Updating dependencies...
  Downloading scikit-fmm-2021.2.2.tar.gz (419 kB)
Collecting scikit-image==0.18.1
  Downloading scikit_image-0.18.1-cp37-cp37m-manylinux1_x86_64.whl (29.2 MB)
Collecting scikit-mpe==0.2.2
  Downloading scikit_mpe-0.2.2-py3-none-any.whl (13 kB)
INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of scikit-image to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of scikit-fmm to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install scikit-fmm==2021.2.2 and scikit-mpe==0.2.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested scikit-fmm==2021.2.2
    scikit-mpe 0.2.2 depends on scikit-fmm<2020.0.0 and >=2019.1.30

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies

Error: failed to update dependecies: error on one or more dependencies, no dependencies were added, see output for details

```

I'm deploying my project to Deta

**The conflict is caused by:
    The user requested scikit-fmm==2021.2.2
    scikit-mpe 0.2.2 depends on scikit-fmm<2020.0.0 and >=2019.1.30**

Could you please bump the dependency version to its latest 2021.2.2 for scikit-fmm?

thanks

**Comment by espdev** [comment]
> scikit-mpe 0.2.2 depends on scikit-fmm<2020.0.0 and >=2019.1.30

The current version of scikit-mpe is 0.2.3 and it depends on `scikit-fmm = ">=2021.1.21,<2021.2"`
Could you try to update scikit-mpe to the latest version?

**Comment by franva** [comment]
hi @espdev 

Thanks for your reply.

As you highlighted, the newer version scikit-mpe is 0.2.3 which is still require older version of fmm which is older than `2021.2`

However, the bug in fmm was only fixed on the 2021.2.2.

**Comment by espdev** [comment]
@franva 

Please try: https://pypi.org/project/scikit-mpe/0.2.4/
